arcaflow_plugin_rtla/rtla_plugin.py

```diff
@@ -181,7 +213,67 @@ def run_timerlat(
                 total_usr_latency[label] = line_list[3]
 
         # Provide the rtla command formatted data as debug output
-        print(output)
+        print(timerlat_output)
+
+        # Example time series output from the tracer (truncated)
+        # <idle>-0      [009] d. 123.769498: #1002 context    irq timer_latency  458 ns
+        #  <...>-625890 [009] .. 123.769499: #1002 context thread timer_latency 666 ns
+        #  <...>-625890 [009] .. 123.769500: #1002 context user-ret timer_latency 499 ns
+        # <idle>-0      [002] d. 123.769528: #1003 context    irq timer_latency  587 ns
+        #  <...>-625883 [002] .. 123.769532: #1003 context thread timer_latency 712 ns
+        #  <...>-625883 [002] .. 123.769534: #1003 context user-ret timer_latency 462 ns
+
+        if params.enable_time_series and trace_file_exists:
+            timeseries_lines = iter(timeseries_output.splitlines())
+
+            # The calculation of the offset from uptime to current time certainly means
+            # that our time series is not 100% accurately aligned to the nanosecond, but
+            # the relative times will be accurate. We'll accept this as good enough.
+            uptime_offset = time.time() - time.monotonic()
```

Comment:
This seems like a very weird operation:  you're calculating the difference between the outputs of two, possibly-independent clocks -- this could be zero or negative -- it would be good to explain here _why_ you're doing this, as well as mentioning possible alignment problems.

Reply:
In terms of the generic/standard definitions of "clock monotonic" this is certainly true; whether the actual relationships of `time.time` and `time.monotonic` on RHEL 8/9/10 provides additional guarantees that make this operation reasonable is a different matter...

Reply:
I'm guessing that Dustin thinks that the trace was produced using the monotonic clock and that he's trying to convert the timestamps to "real (human) time".  But, if that's the case, it would be good to explain that here.